compiler/src/formatting/format.re

```diff
@@ -1773,15 +1789,31 @@ and print_infix_application =
         Doc.concat([Doc.nil, line_comment_docs, Doc.line]);
       };
 
-    Doc.concat([
-      lhs,
-      Doc.space,
-      Doc.text(function_name),
-      trailing_line_comments,
-      after_comments_docs,
-      force_break_if_line_comment(~separator=Doc.nil, after_comments),
-      rhs,
-    ]);
+    if (true) {
```

Comment:
🤔

Reply:
Nooo.    My commit has  picked up my WIP

Let me force remove the commit